op/model.py

```diff
@@ -88,6 +119,32 @@ def __init__(self, name, backend, cache):
 
         self._is_our_unit = self.name == self._backend.unit_name
 
+        self._status = None
```

Comment:
The two empty lines above would look better packed as well. The _app_ attribute may be left as-is if you feel like it, as it has a slightly distinct nature to it.